recipes-kernel/linux/linux-beaglev/0001-sync-beaglev-dts-from-u-boot.patch

```diff
@@ -0,0 +1,1122 @@
+From a8be849b25db468101358c4e907db85c05db5e96 Mon Sep 17 00:00:00 2001
+From: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
+Date: Fri, 30 Apr 2021 23:53:33 +0200
+Subject: [PATCH] sync beaglev dts from u-boot
+
+Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
```

Comment:
Can you add why we need this?

Reply:
@alistair23 The starfive linux tree does not provide (yet?) dts files for BeagleV. Only U-Boot tree does.

Reply:
added a commend inside a patch file